services/galley/test/integration/API.hs

```diff
@@ -242,7 +241,6 @@ tests s =
               test s "existing has password, requested has password - 409" postCodeWithPasswordExistsWithPasswordRequested
             ],
           test s "remove user with only local convs" removeUserNoFederation,
-          test s "remove user with local and remote convs" removeUser,
```

Comment:
Where did these tests go? I don't see them being added to the integration package. Were they already ported?

Reply:
As far as I could see, these tests are redundant. All of the functionality (except the assertions on federated RPCs) is being tested in the new integration.